### PR TITLE
Add configuration, more skeleton

### PR DESCRIPTION
**Pull request by hariso**
### Quick checks:

- [x] There is no other [pull request](https://github.com/conduitio-labs/conduit-connector-databricks/pulls) for the same update/change.
- [ ] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,23 +2,20 @@
...
 generate:
 	go generate ./...
 
-install-paramgen:
-	go install github.com/conduitio/conduit-connector-sdk/cmd/paramgen@latest
+download:
+	@echo Download go.mod dependencies
+	@go mod download
+
+install-tools: download
```
Comment (maha-hajja) [inline Makefile:18]: we should add `install-tools` and `download` to `.PHONY`

File: destination.go [hunk 2]
```diff
@@ -9,41 +24,42 @@ import (
 	sdk "github.com/conduitio/conduit-connector-sdk"
 )
 
+type Config struct {
+	// DSN is a data source name connection string
+	// https://docs.databricks.com/dev-tools/go-sql-driver.html#connect-with-a-dsn-connection-string
+	Token   string `json:"token" validate:"required"`
```
[the remark this answers is not in the document]
Comment (hariso) [reply destination.go:30]: Yup, outdated comment (I started with DSN first, but wasn't flexible enough).

File: paramgen_config.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+		},
+		"httpPath": {
+			Default:     "",
+			Description: "",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{
+				sdk.ValidationRequired{},
+			},
+		},
+		"port": {
```
[the remark this answers is not in the document]
Comment (hariso) [reply paramgen_config.go:28]: Good question, there actually is. Databricks Cloud and Google Cloud use 443, but Azure uses 15001. So I can maybe put 443 to start with.

File: destination.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (hariso) [reply destination.go:30]: Now that I went to add some, to be honest, I don't think I can make it more descriptive. The token is simply the access token, the port is the port, and so on. Does this affect anything?

[the remark this answers is not in the document]
Comment (maha-hajja) [reply destination.go:30]: not really no! I think I prefer to have any description even if it's obvious, ex: (port to connect to databricks), but that's a personal preference so not that important!

File: paramgen_config.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (hariso) [reply paramgen_config.go:28]: It looks like I was wrong, it's Databricks Connect on Azure (and generally) which runs on 15001. So with that having the default set to 443 makes even more sense. Thanks for pointing this out!

File: destination.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (hariso) [reply destination.go]: It does look prettier with something, even if it's obvious. I've added it in the latest commit.

File: destination.go [hunk 4]
```diff
@@ -9,41 +24,44 @@ import (
 	sdk "github.com/conduitio/conduit-connector-sdk"
 )
 
+type Config struct {
+	// Personal access token.
+	Token string `json:"token" validate:"required"`
+	// Databricks server hostname
+	Host string `json:"host" validate:"required"`
+	// Databricks port
+	Port int `json:"port" validate:"required" default:"443"`
```
Comment (maha-hajja) [inline destination.go:33]: we can get rid of the required validation now 👀
```suggestion
	Port int `json:"port" default:"443"`
```

**Review by maha-hajja — APPROVED** [review]

**Pull request merged**